packages/trader/src/Assets/Trading/Categories/Description/accumulator-trade-description.tsx

```diff
@@ -6,22 +6,22 @@ const AccumulatorTradeDescription = ({ onClick }: { onClick: React.MouseEventHan
     const content = [
         <Localize
             i18n_default_text='Accumulators allow you to express a view on the range of movement of an index and grow your stake exponentially at a fixed <0>growth rate</0>.'
-            components={[<span className='contract-type-info__content-definition' onClick={onClick} key={0} />]}
+            components={[<button className='contract-type-info__content-definition' onClick={onClick} key={0} />]}
```

Comment:
weird that sonarcloud would complain about this. Span makes a lot more sense than a button

Reply:
Not sure what complain was but I'd assume it encourages to use onClick event on block elements, which button is and span isn't. It would probably omit the warning if there was, let's say a link instead of a click event.

Reply:
@henry-deriv @akmal-deriv this issue popped-up during TS migration ([6th branch](https://github.com/binary-com/deriv-app/pull/10018)). Sonar cloud started to complain, that '**Visible, non-interactive elements with click handlers must have at least one keyboard listener'** (to increase accessibility) even though I didn't change the code (except TS). After slack discussion I was suggested to replace `span`/`div` with more semantic tags (like `button`) or/and to add keyboard listener. In order not to block 6th package I created a separate card for it.